templates/zerver/emails/confirm_registration.source.html

```diff
@@ -20,5 +20,11 @@
     {% endif %}
     <a class="button" href="{{ activate_url }}">{{ _('Complete registration') }}</a>
 </p>
-<p>{% trans support_email=macros.email_tag(support_email) %}Contact us any time at {{ support_email }} if you run into trouble, have any feedback, or just want to chat!{% endtrans %}</p>
+<p>
+    {% if realm_plan_type==1 %}
+        {{macros.contact_us_self_hosted(server_admin_email)}}
+    {% else %}
+        {{macros.contact_us_cloud(support_email)}}
+    {% endif %}
+</p>
```

Comment:
I think you should continue to use the `{% trans support_email=macros.email_tag(support_email) %} {% endtrans %}` tags. This helps convert the email into a clickable link.

Reply:
I have made it a clickable link in the macro
```

{% macro contact_us_self_hosted(email) -%}
    If you have any questions, please contact this Zulip server's administrators at <a href="{{ email }}">{{ email }}</a>.
{%- endmacro %}
```
I am making it a macro since it was mentioned in the issue.
I think I should keep it, however I now noticed I did miss the trans tag